TeslaSolarCharger.Model/Entities/TeslaSolarCharger/Car.cs

```diff
@@ -15,6 +15,7 @@ public class Car
     public DateTime LatestTimeToReachSoC { get; set; }
 
     public bool IgnoreLatestTimeToReachSocDate { get; set; }
+    public bool IgnoreLatestTimeToReachSocDateOnWeekdays { get; set; }
```

Comment:
Shouldn't all these variables, migrations and methods be called IgnoreOnWeekends

Reply:
I Will change it